src/euphorie/upgrade/deployment/v1/20211216172426_toggle_for_self_registration/registry.xml

```diff
@@ -0,0 +1,11 @@
+<registry xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+  <record name="plone.enable_self_reg" interface="Products.CMFPlone.interfaces.controlpanel.ISecuritySchema" field="enable_self_reg">
+    <field type="plone.registry.field.Bool">
```

Comment:
The field element can be omitted if I am not wrong, given that Plone should already have it.